strutil/quantity/quantity.go

```diff
@@ -23,9 +23,35 @@ import (
 	"fmt"
 	"math"
 
-	"github.com/snapcore/snapd/i18n"
+	"github.com/canonical/golib/i18n"
 )
 
+// Common package strings with translation support
+//
+// Note: This may not be converted as part of global 'const' or 'var'
+// variables, and also not inside 'init()'. The i18n implementation
+// is only available after early initialisation.
+
+func secs() rune {
```

Comment:
Is it necessary to change all of these to returning `rune` rather than `string`? Reverting back to string would also avoid the `%s` to `%c` changes below.

Reply:
The formatter total output length may never exceed 5 runes. This is a hard requirement for code such as the ANSI meter progress bar implementation.